frontend/src/layout/navigation/TopBar/announcementLogic.test.ts

```diff
@@ -20,23 +20,33 @@ describe('announcementLogic', () => {
         logic.mount()
         await expectLogic(logic).toMount([featureFlagLogic, preflightLogic, userLogic, navigationLogic])
         featureFlagLogic.actions.setFeatureFlags([FEATURE_FLAGS.CLOUD_ANNOUNCEMENT], {
-            [FEATURE_FLAGS.CLOUD_ANNOUNCEMENT]: MOCK_CLOUD_ANNOUNCEMENT,
+            [FEATURE_FLAGS.CLOUD_ANNOUNCEMENT]: true,
         })
     })
 
     afterEach(() => logic.unmount())
     it('shows a cloud announcement', async () => {
         await expectLogic(logic).toMatchValues({
-            cloudAnnouncement: MOCK_CLOUD_ANNOUNCEMENT,
+            cloudAnnouncement: DefaultCloudAnnouncement,
             shownAnnouncementType: AnnouncementType.CloudFlag,
         })
     })
 
     it('hides announcements during the ingestion phase', async () => {
         router.actions.push(urls.ingestion())
         await expectLogic(logic).toMatchValues({
-            cloudAnnouncement: MOCK_CLOUD_ANNOUNCEMENT,
+            cloudAnnouncement: DefaultCloudAnnouncement,
             shownAnnouncementType: null,
         })
     })
+
+    it('shows a customised cloud announcement from payload', async () => {
+        featureFlagLogic.actions.setFeatureFlags([FEATURE_FLAGS.CLOUD_ANNOUNCEMENT], {
+            [FEATURE_FLAGS.CLOUD_ANNOUNCEMENT]: true, // not sure how to add payload
```

Comment:
@neilkakkar  help 😅

Reply:
Yeah our hand-rolled logic doesn't support payloads, it comes straight from `posthog-js`, so you'll need to mock it. Something like:

```
jest.mock('posthog-js', () => ({
  getFeatureFlagPayload: jest.fn(),
}));
```

Reply:
ok, I'd just remove the test in that case